robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Disallow: /admin/
+Disallow: /private/
```

Comment:
Question: Is Disallow mandatory in a robots.txt file?

Just from reading this it appears we are disallowing 2 sub-parts of our site:
1) /admin/
2) /private/

We do not currently have either of those on our site at the moment. 
So we should not care to add Disallow statements for things that do not exist.

If we need to include a Disallow statement, let's apply it to something that exists.